### PR TITLE
Update `torchaudio` to 0.9 for `tensorflow-gpu` compatibility

### PR DESCRIPTION
**Pull request by jbitton**
Summary:
As called out in https://github.com/facebookresearch/AugLy/issues/28, there are some conflicting dependencies between `torchaudio`/`torch` 0.8.1/1.8.1 and `tensorflow-gpu`.

However, as discovered in https://github.com/pytorch/audio/issues/1595, upgrading to v0.9 etc actually resolve this issue.

Thus, I update the torchaudio/torch versions in our `requirements.txt` and I also updated our `numpy` requirement so there are no conflicting dependencies between `tf-gpu` and `augly` :)

I verified on my side that all unit tests still pass and that `setup.py` finishes as expected with no errors. I also update `setup.py` to add our README to our PyPI page.

Ran:
```
python -m unittest augly.tests.audio_tests.functional_unit_tests
----------------------------------------------------------------------
Ran 21 tests in 4.683s

OK

python -m unittest augly.tests.audio_tests.transforms_unit_tests
----------------------------------------------------------------------
Ran 24 tests in 1.892s

OK

python -m unittest augly.tests.image_tests.functional_unit_tests
----------------------------------------------------------------------
Ran 32 tests in 32.442s

OK (skipped=2)

python -m unittest augly.tests.image_tests.transforms_unit_tests
----------------------------------------------------------------------
Ran 39 tests in 12.182s

OK (skipped=3)

python -m unittest augly.tests.video_tests.transforms.cv2_tests
----------------------------------------------------------------------
Ran 5 tests in 72.686s

OK
```

and

```
python setup.py install
.....
Using /home/jbitton/anaconda3/envs/testing_reqs/lib/python3.9/site-packages
Finished processing dependencies for augly==0.1.1
```

Differential Revision: D29292956

### CODE REVIEW

**Comment by jbitton** [comment]
@zpapakipos let me know if running this on your side works without issue. if yes, let's merge + update the PyPI package!

**Comment by zpapakipos** [comment]
Tested on my side.

Installed using new reqs locally:
```bash
cd AugLy/
git pull
git checkout change_reqs
conda create -n augly_change_reqs -y
conda activate augly_change_reqs
cd ..
pip install -e AugLy/
```

Ran unit tests:
```bash
python -m unittest augly.tests.audio_tests.functional_unit_tests
----------------------------------------------------------------------
Ran 21 tests in 2.664s

OK

python -m unittest augly.tests.audio_tests.transforms_unit_tests
----------------------------------------------------------------------
Ran 24 tests in 1.069s

OK

python -m unittest augly.tests.image_tests.functional_unit_tests
----------------------------------------------------------------------
Ran 32 tests in 27.815s

OK (skipped=2)

python -m unittest augly.tests.image_tests.transforms_unit_tests
----------------------------------------------------------------------
Ran 39 tests in 10.127s

OK (skipped=3)

python -m unittest augly.tests.text_tests.functional_unit_tests
----------------------------------------------------------------------
Ran 12 tests in 0.059s

OK

python -m unittest augly.tests.text_tests.transforms_unit_tests
----------------------------------------------------------------------
Ran 12 tests in 0.016s

OK

python -m unittest augly.tests.video_tests.transforms.composite_tests
----------------------------------------------------------------------
Ran 11 tests in 31.113s

OK
```

Built & uploaded new pypi package:
```bash
<changed version # in setup.py to 0.1.2>
git add setup.py
git commit -m "Update version # to 0.1.2"
git push
conda activate pypi
python -m build --wheel
twine upload dist/*
```

Tested new pypi package (in colab audio example notebook):
```python
!pip install -U augly
!sudo apt-get install python3-magic

import os
import augly.audio as audaugs
import augly.utils as utils
from IPython.display import display, Audio

# Get input audio
input_audio = os.path.join(
    utils.TEST_URI, "audio", "inputs", "vad-go-stereo-44100.wav"
)
display(Audio(input_audio))

from augly.audio.utils import validate_and_load_audio

input_audio_arr, sr = validate_and_load_audio(input_audio)
meta = []
aug = audaugs.PeakingEqualizer()
aug_audio, sr = aug(input_audio_arr, sample_rate=sr, metadata=meta)
display(Audio(aug_audio, rate=sr))
meta
```

File: setup.py [hunk 1]
```diff
@@ -3,20 +3,24 @@
...
     if '@' not in r
 ]
 
+with open("README.md", encoding="utf8") as f:
+    readme = f.read()
+
 
 setuptools.setup(
     name="augly",
     version="0.1.1",
```
Comment (zpapakipos) [inline setup.py:20]:
```suggestion
    version="0.1.2",
```

**Review by zpapakipos — APPROVED** [review]
Please just update the version # in setup.py since I changed that in the `change_reqs` branch before uploading the package, but other than that looks great, and all my testing works! Thanks for making this change :)

**Pull request merged**